app/src/main/java/org/sopt/and/MyActivity.kt

```diff
@@ -0,0 +1,123 @@
+package org.sopt.and
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.User.EMAIL
+import org.sopt.and.component.MyPageBox
+import org.sopt.and.component.MyPagePurchaseBox
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val intent = intent
+        val email = intent.getStringExtra(EMAIL) ?: ""
+        setContent {
+            ANDANDROIDTheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    MyPageScreen(
+                        modifier = Modifier.padding(innerPadding),
+                        email = email
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MyPageScreen(modifier: Modifier, email: String) {
```

Comment:
modifier는 일반적으로 변수를 미리 할당해둬요~ 옵셔널로 하는게 좋을 것 같습니다!
modifier는 추가적인 변경이 필요할 경우에만 넣어주기 때문에 디폴트를 넣어주구요, 이렇게 하면 preview에서 안써도 된답니당
```suggestion
fun MyPageScreen(
    email: String,
    modifier: Modifier = Modifier
) {
```